sympy/tensor/array/tests/test_array_derivatives.py

```diff
@@ -48,5 +48,5 @@ def test_array_derivative_construction():
     d = ArrayDerivative(M.as_explicit(), (N.as_explicit(), 2), evaluate=False)
     assert d.doit().shape == (4, 3, 4, 3, 3, 2)
     expr = d.doit()
-    assert isinstance(expr, ArrayDerivative)
+    assert isinstance(expr, NDimArray)
```

Comment:
I wonder how does the result change after this.
Is it printed as explicit matrix now?

Reply:
Yes.

Reply:
I would like to see how the content looks like. But it can be huge as much as the dimension tells.